cfgov/cfgov/urls.py

```diff
@@ -528,6 +530,10 @@ def empty_200_response(request, *args, **kwargs):
 ]
 urlpatterns = urlpatterns + category_redirects
 
+if flag_enabled('ENABLE_WAGTAIL_API'):
```

Comment:
I wasn't able to use `flagged_url` for this because `django-flags` won't accept a set of URLs in the format the wagtail API provides. I discussed this with @willbarton and he suggested this alternative.

Reply:
This approach means that the `flag_enabled` call checks the database any time urls.py is imported. This could cause problems if, say, that table doesn't exist because you're running off of [a fresh database](http://cfpb.github.io/cfgov-refresh/installation/#install-postgres) (see https://github.com/cfpb/django-flags/issues/18). This can be reproduced like this:

```
$ dropdb --if-exists cfgov && dropuser --if-exists cfpb
$ createuser cfpb && createdb -O cfpb cfgov
$ psql postgres://cfpb@localhost/cfgov -c 'CREATE SCHEMA cfpb'
$ cfgov/manage.py migrate
# This should create DB tables, but fails, because the import checks the DB first:
...
  File "/Users/chosaka/Projects/cfgov-refresh/cfgov/cfgov/urls.py", line 533, in <module>
    if flag_enabled('ENABLE_WAGTAIL_API'):
...
django.db.utils.ProgrammingError: relation "flags_flagstate" does not exist
LINE 1: ...state"."value", "flags_flagstate"."required" FROM "flags_fla...
```

This could also causes a problem if we ever want to run code that imports Django URLs even when there isn't a Django database configured:

```
# shut down Postgres
$ brew services stop postgresql@10
# on master, this works
$ cfgov/manage.py shell
>>> import cfgov.urls
>>>
# but not on this branch, because the import hits the DB
>>> import cfgov.urls
django.db.utils.OperationalError: could not connect to server: Connection refused
        Is the server running on host "localhost" (::1) and accepting
        TCP/IP connections on port 5432?
could not connect to server: Connection refused
        Is the server running on host "localhost" (127.0.0.1) and accepting
        TCP/IP connections on port 5432?
```

As to how big of a problem this actually is, I'm not sure. Is there a way to use some kind of `ignore_errors` logic as in https://github.com/cfpb/django-flags/pull/20 to handle these cases?

Reply:
Hmm, that's unfortunate. Maybe I misremembered Will's suggestion. @willbarton What strategy did you recommend for adding the admin URL paths when `flagged_url` doesn't work?